### DISCUSSION TITLE
option to ignore same bird for X minutes

### DISCUSSION THREAD

**Discussion by waynecarnold** [mcguirepr89/BirdNET-Pi#957] [Ideas]
I have a few species that chirp non stop for most of the day. The result is a daily count of 500+ when it is in reality only one or two birds chirping continuously. This is filling my disk pretty quickly. If there was a setting to ignore a second detection of same bird for X amount of time, that would be nice.

Love the app and not being a coder I can only request others to do the work, sorry :)

Wayne

**Comment by kirbybrianj** [comment]
There are a bunch of ways one could think in the birdnet repo to throttle a very active species (for me, the gray catbirds which happily vocalize 5000 times a day), including wait for X minutes, only allow N times per M minutes, or put onto excluded species list F fraction of the time. I'd be happy to help implement this although I have never looked at the code and I would be unlikely to succeed quickly without help from someone who has worked on it, and if I didn't succeed quickly I'd soon be pulled away to other things.

Experimenting with Cunningham's Law, I say post calls to birdweather, use birdweather's API to query whether a bird has called in last X minutes, then use that to toggle birdnet-pi's "excluded species" status of the bird.
